cmd/flowserver/server_test.go

```diff
@@ -608,24 +609,24 @@ func (ts *ServerTestSuite) TestFlowStartAndResume() {
 
 func (ts *ServerTestSuite) TestWebhookMocking() {
 	testCases := []struct {
-		config         string
-		expectedStatus int
-		expectedBody   string
+		config        string
+		expectedValue string
+		expectedExtra string
 	}{
 		// default config is make webhook calls
-		{`{}`, 200, `{ "ok": "true" }`},
+		{`{}`, `200`, `{"ok":"true"}`},
 
 		// explicitly disabled or enabled
-		{`{"disable_webhooks": false}`, 200, `{ "ok": "true" }`},
-		{`{"disable_webhooks": true}`, 200, "DISABLED"},
+		{`{"disable_webhooks": false}`, `200`, `{"ok":"true"}`},
+		{`{"disable_webhooks": true}`, `200`, `"DISABLED"`},
```

Comment:
if webhook returned non-JSON text, we convert it to a JSON string so it's still JSON